docs/_articles/en/apex/interactive-debugger.md

```diff
@@ -98,7 +98,7 @@ Add filters to the `"Launch Apex Debugger"` configuration:
 
 To auto-complete potential request type values for `"requestTypeFilter"`, press Ctrl+Space.
 
-To filter by entry point, enter a regular expression as the value for `"entryPointFilter"`. For example, to whitelist requests made by the Visualforce page `MyPage`, enter `".*/apex/MyPage.apexp"`.
+To filter by entry point, enter a regular expression as the value for `"entryPointFilter"`. For example, to allowlist requests made by the Visualforce page `MyPage`, enter `".*/apex/MyPage.apexp"`.
```

Comment:
Yes, this does sound funny now that I re-read it. I'm fine if you keep as-is. But will provide this suggestion: remove the "list" - how about? For example, to allow requests made by...

Reply:
Done